io.catenax.material_flow_simulation_result/3.0.0/MaterialFlowSimulationResultAspect.ttl

```diff
@@ -0,0 +1,336 @@
+   samm:characteristic :AmountCharacteristic ;
+   samm:exampleValue "1"^^xsd:positiveInteger .
+
+:batches a samm:Property ;
+   samm:preferredName "Batches"@en ;
+   samm:description "List of material batches contained within a handling unit"@en ;
+   samm:characteristic :BatchList .
+
+:VolumeCharacteristic a samm-c:Measurement ;
+   samm:preferredName "Volume Characteristic"@en ;
+   samm:description "Volumetric value determining the volume of the handling unit"@en ;
+   samm:dataType xsd:float ;
+   samm-c:unit unit:litre .
+
+:WeightCharacteristic a samm-c:Measurement ;
+   samm:preferredName "Weight Characteristic"@en ;
+   samm:description "Amount of weight of the handling unit"@en ;
+   samm:dataType xsd:float ;
+   samm-c:unit unit:kilogram .
+
+:AmountCharacteristic a samm-c:Quantifiable ;
+   samm:preferredName "Amount Characteristic"@en ;
+   samm:description "Number describing handling units with identical pieces"@en ;
+   samm:dataType xsd:positiveInteger ;
+   samm-c:unit unit:piece .
+
+:BatchList a samm-c:List ;
+   samm:preferredName "Batch List"@en ;
+   samm:description "List of material batches"@en ;
+   samm:dataType :MaterialBatch .
+
+:MaterialBatch a samm:Entity ;
+   samm:preferredName "Material Batch"@en ;
+   samm:description "Material Batches are part of a handling unit"@en ;
+   samm:properties ( :batchSerialNumber :materialNumber :materialName :materialHazardousGoods :batchExpirationTimestamp :quantity :unitOfMeasurement :batchNumber :batchOrderId ) .
+
+:batchSerialNumber a samm:Property ;
+   samm:preferredName "Batch Serial Number"@en ;
+   samm:description "can be used to uniquely identify a singe product part"@en ;
+   samm:characteristic :Text ;
+   samm:exampleValue "Batch_1" .
+
+:materialNumber a samm:Property ;
+   samm:preferredName "Material Number"@en ;
+   samm:description "ID of the material contained in the batch"@en ;
+   samm:characteristic :Text ;
+   samm:exampleValue "KK1000GR" .
+
+:materialName a samm:Property ;
+   samm:preferredName "Material Name"@en ;
+   samm:description "Name of the material"@en ;
+   samm:characteristic :Text ;
+   samm:exampleValue "KK1000GR-Gehäuse-Rot" .
+
+:materialHazardousGoods a samm:Property ;
+   samm:preferredName "Material Hazardous Goods"@en ;
+   samm:description "Flags hazardous goods"@en ;
+   samm:characteristic samm-c:Boolean ;
+   samm:exampleValue false .
+
+:batchExpirationTimestamp a samm:Property ;
+   samm:preferredName "Batch Expiration Timestamp"@en ;
+   samm:description "Expiration date of batch"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2023-08-22T16:00:00.000Z"^^xsd:dateTime .
+
+:quantity a samm:Property ;
+   samm:preferredName "Quantity"@en ;
+   samm:description "Amount of materials"@en ;
+   samm:characteristic :QuantityCharacteristic ;
+   samm:exampleValue "50"^^xsd:float .
+
+:unitOfMeasurement a samm:Property ;
+   samm:preferredName "UnitOfMeasurement"@en ;
```

Comment:
UnitOfMeasurement should not be together

Reply:
pefered name has been changed